src/vg_canvas.ml

```diff
@@ -1,5 +1,6 @@
 open! Base
 open! Virtual_dom.Vdom
+open Js_of_ocaml
```

Comment:
I'd rather not have an open here.  Can we just locally qualify where we use Js_of_ocaml?

Reply:
Done